docs/products/postgresql/howto/pg-long-running-queries.rst

```diff
@@ -81,4 +81,4 @@ which you should see the role::
     
 To resolve the permission issue, you may grant the user the appropriate role as per below::
 
-    grant testrole to avnadmin;
```

Comment:
was this something you changed or maybe github is showing it wrong?

Reply:
It's got a `+` on the next line for the same text, and I believe the only difference is adding in a newline at the end of the last line of the file.
GitHub doesn't make it very obvious, but `gitk`, for instance, mentions it explicitly.
I'm torn about a change like this - it's an incidental change, caused by my editor, but honestly all lines in a file should end with a new line.
(This matters more on Unix than on other systems - see https://medium.com/@alexey.inkin/how-to-force-newline-at-end-of-files-and-why-you-should-do-it-fdf76d1d090e for some background)

Reply:
I agree with you that all lines should end with a new line, so thanks for clarifying this behavior.